BusinessLogic/Services/ToDoDataService.cs

```diff
@@ -16,51 +16,31 @@ public ToDoDataService(PortfolioContext _context)
             context = _context;
         }
 
-        //    using (var context = _contextFactory.CreateDbContext())
-        //    {
-        //        // ...
-        //    }
-
         public void InitialiseDb()
         {
             if (context.ToDoLists.Any())
             {
                 return;
             }
 
-            ToDoList list1 = new ToDoList { Name = "Work Items" };
-            ToDoList list2 = new ToDoList { Name = "Shopping List" };
+            ToDoList list1 = new ToDoList { Title = "Urgent + Important" };
```

Comment:
You should be able to shrink this to something like

var list= new ToDoList("Title", new List<ToDoItem>(){ new ToDoItem("title")})
But ofcourse spread out over a couple of lines. to look nicer.
This will read easier, because youve grouped the item that owns things, and the things its owned.

Also, unless your items are DTO's (Data transfer objects), always prefer constructors.